fuzz/fuzzers/decode.rs

```diff
@@ -0,0 +1,16 @@
+#![no_main]
+#[macro_use] extern crate libfuzzer_sys;
+extern crate rawloader;
+
+use std::io;
+
+fuzz_target!(|data: &[u8]| {
```

Comment:
`decode_safe()` takes a file, not an `io::Read`, so it cannot use the fuzzer-provided `&[u8]` without writing that to a file first. I wrapped the `decode()` in a a `panic::catch_unwind` instead.